dataplicity/m2m/wsclient.py

```diff
@@ -310,6 +313,16 @@ def connect(self, wait=True, timeout=None):
             return self.wait_ready(timeout=timeout)
         return None
 
+    def abandon(self):
```

Comment:
Reset everything so this object won't process any packets nor block. To fix suspected thread not closing.